MoreLinq/Pairs.cs

```diff
@@ -0,0 +1,65 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2012 Atif Aziz. All rights reserved.
```

Comment:
Copyright notice is incorrect. Please correct the year and author.

Reply:
Great point. I'll remember this.